<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CabinetViewModel</class>
 <widget class="QWidget" name="CabinetViewModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="profileDataButton">
         <property name="text">
          <string>Личные данные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="activeOrdersButton">
         <property name="text">
          <string>Активные заказы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closedOrdersButton">
         <property name="text">
          <string>Завершенные заказы</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
